homework/index.js

```diff
@@ -93,18 +116,21 @@
         cellContent.textContent = repo[content[i].attribute];
       }
     }
-    fetchJSON(repo.contributors_url, (error, contributors) => renderContributors(contributors));
+
+    fetchJSON(repo.contributors_url)
+      .then(contributors => renderContributors(contributors))
+      .catch(error => {
+        console.error(error);
```

Comment:
You should render errors to the page, not to the console. Regular users do not run your app with the Chrome console open, so they will not see this error.

Reply:
i IMPLEMENT IT